CMakeLists.txt

```diff
@@ -33,22 +33,23 @@ install(EXPORT ArborXTargets
   DESTINATION lib/cmake/ArborX
 )
 
-SET(ARBORX_VERSION_STRING "0.9 (dev)")
+SET(ARBORX_VERSION_STRING "0.9-dev)")
```

Comment:
With this change we will print `0.9-dev)`, that's pretty ugly. Also in this file, we have mainly used lowercase keywords and not uppercase ones.